main.js

```diff
@@ -0,0 +1,290 @@
+                    acceptPlayerNameTwo.hidden = true;
+                    acceptPlayerNameOne.hidden = true;
+                    startPlay(cellArray)
+                    playerTwo = 'Computer'
+                }
+            })
+
+    })
+
+// Two player start button functionality
+twoPlayerStart.addEventListener('click',
+    function () {
+        playerInputDiv.hidden = false
+        acceptPlayerNameTwo.hidden = true
+        acceptPlayerNameOne.hidden = false
+        onePlayerStart.disabled = true;
+        twoPlayerStart.disabled = true;
+        //event listener for plugging in first player name and giving prompt for second player input
+        acceptPlayerNameOne.addEventListener('click',
+            function () {
+                if (playerName.value === '') {
+                    playerOne = 'X'
+                    playerName.value = ''
+                    playerInputOption.textContent = 'Two'
+                    acceptPlayerNameTwo.hidden = false
+                    acceptPlayerNameOne.hidden = true
+                } else {
+                    playerOne = playerName.value
+                    playerName.value = ''
+                    playerInputOption.textContent = 'Two'
+                    acceptPlayerNameTwo.hidden = false
+                    acceptPlayerNameOne.hidden = true
+                }
+
+            })
+        //Event Listener for plugging in second player name
+        acceptPlayerNameTwo.addEventListener('click',
+            function () {
+                if (playerName.value === '') {
+                    playerTwo = 'O'
+                    startPlay(cellArray)
+                } else {
+                    playerTwo = playerName.value
+                    startPlay(cellArray)
+                }
+            })
+    })
+
+// restart button functionality
+restartGame.addEventListener('click',
+    function () {
+        playerInputOption.textContent = ''
+        if (onePlayerStart.disabled === true) {
+            onePlayerStart.disabled = false;
+            twoPlayerStart.disabled = false;
+            playerTurnDiv.hidden = true;
+            playerInputDiv.hidden = true;
+            timerDiv.hidden = true
+            // set cells to empty strings
+            cellArray.forEach(function (cellText) {
+                cellText.textContent = ''
+            })
+            // set css decorations to blank
+            cellArray.forEach(function (cellClass) {
+                cellClass.className = ''
+            })
+            // remove ability to click on cells
+            removeFillSquare(event)
+            // restart timer
+            clearInterval(secondCounter)
+            seconds = 00
+            gameTimer.textContent = seconds
+            whoseTurnIsIt.textContent = ''
+            winnerTitle.textContent = ''
+            playerName.value = ''
+            usedCellArray = []
+        }
+        // if game isnt started, clicking this will do nothing
+        else { }
```

Comment:
If you want to do something only when a certain thing happens and then do nothing otherwise, you should be able to omit the else and it will achieve the same effect.